### PR TITLE
Spring cleanup

### PR DESCRIPTION
**Pull request by FilipDolnik**

### CODE REVIEW

File: Cuckoo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -245,7 +340,7 @@
...
+			shellScript = "# Find the installed Cuckoo version\nCUCKOO_VERSION=\"0.4.0\"\n\n# Used to ignore Xcode's environment\nalias run=\"env -i PATH=$PATH, HOME=$HOME\"\n\ncuckoo runtime --check $CUCKOO_VERSION\ncuckooReturn=$?\n\nif [ $cuckooReturn == 1 ]; then\n# Update local brew repository and upgrade to latest Cuckoo generator\nrun brew update --verbose\nrun brew upgrade --verbose SwiftKit/cuckoo/cuckoo\nelif [ $cuckooReturn == 127 ]; then\n# Update local brew repository and install latest Cuckoo generator\nrun brew install --verbose SwiftKit/cuckoo/cuckoo\nfi\n\n#cuckoo generate --runtime $CUCKOO_VERSION --output \"$PROJECT_DIR/Tests/GeneratedMocks.swift\" \"$PROJECT_DIR/Tests/TestedProtocol.swift\" \"$PROJECT_DIR/Tests/TestedClass.swift\"";
```
Comment (TadeasKriz) [inline Cuckoo.xcodeproj/project.pbxproj]: 0.4.1

Comment (TadeasKriz) [inline Cuckoo.xcodeproj/project.pbxproj]: You forgot to not commit the commented out `cuckoo generate` command.

File: Tests/CuckooAPITest.swift [hunk 2]
```diff
@@ -59,12 +59,12 @@ class MockeryAPITest: XCTestCase {
         
         XCTAssertEqual(mock.withReturn(), "hello world!")
         
-        var helloWorld: String = ""
+        /*var helloWorld: String = ""
```
Comment (TadeasKriz) [inline Tests/CuckooAPITest.swift]: If commenting out, you have to add a comment why is that.

File: Source/matching/MatcherFunctions.swift [hunk 3]
```diff
@@ -11,14 +11,14 @@ private func compareCalls(count: Int, whenNil: Bool = false, using function: (In
 }
 
 private func describeCallMismatch(calls: [StubCall], description: Description) {
-    description.appendText("was called ").appendValue(calls.count).appendText(" times")
+    description.append("was called ", calls.count, " times")
```
Comment (TadeasKriz) [inline Source/matching/MatcherFunctions.swift]: Why is this changed? (applies to all uses)

File: Source/matching/Matcher.swift [hunk 4]
```diff
@@ -24,10 +24,6 @@ extension Matcher {
     public var matcher: AnyMatcher<MatchedType> {
         return typeErased()
     }
-    
-    public func describeMismatch(input: MatchedType, to description: Description) {
```
Comment (TadeasKriz) [inline Source/matching/Matcher.swift]: Why is it removed?

**Comment by TadeasKriz** [comment]
Travis still not happy.

File: Source/matching/Matcher.swift (hunk 4 above)
Comment (FilipDolnik) [inline Source/matching/Matcher.swift]: It was only moved to FunctionMatcher, AnyMatcher has its own implementation.

Comment (TadeasKriz) [inline Source/matching/Matcher.swift]: But that's why it was in the protocol extension. Each time you create a different matcher, you don't need to reimplement it.

Comment (FilipDolnik) [inline Source/matching/Matcher.swift]: Not sure if there will be another implementations. But maybe we should consider making one as replacement for AnyMatcher<[StubCall]>, which stands for number of function calls in verify.

Comment (TadeasKriz) [inline Source/matching/Matcher.swift]: Why? AnyMatcher is just a type erasure for other matchers basically. And there might be more matchers in the future, so why making it more difficult?
